redesign/src/404.html

```diff
@@ -0,0 +1,41 @@
+---
+layout: layouts/default.njk
+title: 404 | Expunge Maryland 
+---
+
+{% include "_nav.html" %}
+{% include "_header.html" %}
+
+<section class= "site-wrapper">
+
+    <section class="main error-page">
+        <button class= "back u-button-reset" onclick="goBack()">
+            <p class="back--text"> Back </p>
+                <section class="back--icon">
+                <div class="back--arrow"></div> <hr>
+                </section>
+        </button>
+         
+        <section class="outline-title"> <p class="error--title">404</p> </section>
```

Comment:
```suggestion
        <div class="outline-title error--title">404</div>
```
1. This should probably just be a `div` and not a `section` (see [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section)).
2. I'd explore if you can get your current styles to work without the extra markup, too.
3. If you only use these classnames on this element, you can also probably whittle them down to a single class.